locales/en-US/notes.properties

```diff
@@ -0,0 +1,13 @@
+welcomeTitle=Welcome!
+welcomeText=This is a simple one-page notepad built in to Firefox that helps you get the most out of the web.
+
+emptyPlaceHolder=Take a note…
+
+openingLogin=Opening Login…
+forgetEmail=Forget this Email
+
+syncNotReady=Sorry, Sync is not ready yet, but we will count your click as a vote to hurry it up!
```

Comment:
On a side note, I would really stay away from "Notes Sync", then you need to figure out if it's a brand, if it's localizable, which part given that "Notes" won't be. Basically a can of worms ;-)